test/core/global.wast

```diff
@@ -1,29 +1,45 @@
 ;; Test globals
 
 (module
+  (global (import "spectest" "global_i32") i32)
+  (global (import "spectest" "global_i64") i64)
+
   (global $a i32 (i32.const -2))
-  (global (;1;) f32 (f32.const -3))
-  (global (;2;) f64 (f64.const -4))
+  (global (;2;) f32 (f32.const -3))
+  (global (;3;) f64 (f64.const -4))
   (global $b i64 (i64.const -5))
+  (global $c i32 (i32.add (i32.sub (i32.mul (i32.const 20) (i32.const 2)) (i32.const 2)) (i32.const 4)))
+  (global $d i64 (i64.add (i64.sub (i64.mul (i64.const 20) (i64.const 2)) (i64.const 2)) (i64.const 5)))
 
   (global $x (mut i32) (i32.const -12))
-  (global (;5;) (mut f32) (f32.const -13))
-  (global (;6;) (mut f64) (f64.const -14))
+  (global (;6;) (mut f32) (f32.const -13))
+  (global (;7;) (mut f64) (f64.const -14))
   (global $y (mut i64) (i64.const -15))
 
+  (global $z1 i32 (global.get 0))
```

Comment:
I couldn't find any tests in this file where a global was used to initialize another global.. so I added this simple case.  Am I looking in the wrong place maybe?